packages/espressocash_app/lib/ui/app_bar.dart

```diff
@@ -79,6 +80,9 @@ class _CpAppBarState extends State<CpAppBar> {
 
     return AppBar(
       title: widget.title,
+      titleTextStyle: Theme.of(context).textTheme.labelLarge?.copyWith(
+            color: CpColors.lightGreyBackground,
```

Comment:
It's breaking webview screens, e.g. transaction details screen. It should probably implement some dark theme as well:

<img width="400" src="https://github.com/espresso-cash/espresso-cash-public/assets/853356/1cd8c246-7b59-4416-b1d5-d904c3a13bb7">

Reply:
Removed titleTextStyle from AppBar and applied theme to webviews in [2420692](https://github.com/espresso-cash/espresso-cash-public/pull/1461/commits/2420692a57c0b0623aec623a0a1e2eac6c44bf52)